assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -23,6 +23,7 @@
     HfModelTypes,
     MLFlowHFFlavourConstants,
     LOGS_TO_BE_FILTERED_IN_APPINSIGHTS,
+    SUPPORTED_FLAVORS,
```

Comment:
Can we change the name like MLFLOW_FLAVORS?

Reply:
Fixed